nixos/modules/services/network-filesystems/ipfs.nix

```diff
@@ -226,8 +226,8 @@ in {
             ${optionalString (! cfg.localDiscovery) "--profile=server"}
         else
           ${if cfg.localDiscovery
-            then "ipfs config profile apply local-discovery"
-            else "ipfs config profile apply server"
+            then "ipfs --offline config profile apply local-discovery"
+            else "ipfs --offline config profile apply server"
```

Comment:
Nit: it is probably better to unify these lines and just make the profile itself variable. That also makes it move obvious at a quick glance that we always apply a profile here.